OrbitGl/App.cpp

```diff
@@ -854,6 +854,12 @@ void OrbitApp::SendToUiNow(const std::string& message) {
   }
 }
 
+//-----------------------------------------------------------------------------
+void OrbitApp::SendInfoToUi(const std::string& title, const std::string& text) {
+  std::string message = "info:" + title + "\n" + text;
+  SendToUiNow(message);
+}
+
 //-----------------------------------------------------------------------------
 void OrbitApp::SendErrorToUi(const std::string& title,
                              const std::string& text) {
```

Comment:
we have it twice now.

Reply:
The previous is for an information message box, this is for an error message box.